### ISSUE TITLE
Won't refuse to remove changes

### ISSUE THREAD

**Issue by Gastropied** [desktop/desktop#19511]
### The problem

I'm trying to change branches but it says unable to check out there are changes on your branch. the following would be overwritten.
i try to discard the changes and they won't. No error messages pops up.

### Release version

3.4.8

### Operating system

Windows 11

### Steps to reproduce the behavior

I have no idea.

### Log files

[2024-11-06.desktop.production.log](https://github.com/user-attachments/files/17643512/2024-11-06.desktop.production.log)

**Comment by steveward** [comment]
@Gastropied thanks for the issue. It looks like this is the error:

```
2024-11-06T07:25:59.789Z - error: [ui] `git checkout --progress origin/Dev_Tyler -b Dev_Tyler --recurse-submodules --` exited with an unexpected code: 1.
stderr:
error: Your local changes to the following files would be overwritten by checkout:
	Content/StarterContent/Textures/T_Wood_Walnut_D.uasset
Please commit your changes or stash them before you switch branches.
Aborting
```

Is that `.uasset` file showing as an uncommitted change in GitHub Desktop? If it's an asset file is it's possible that it's open in an external application and unable to be discarded?

**Comment by Gastropied** [comment]
Thank you for responding.
And I tried stashing and discarding the changes but it wouldn't let me. And
I didn't want to commit these changes to the branch as I wasn't aware of
any changes made to that particular asset.

**Comment by steveward** [comment]
Is that file open in Unreal Engine? You can try manually discarding the file from the command line:

`git checkout -- Content/StarterContent/Textures/T_Wood_Walnut_D.uasset`

**Comment by Gastropied** [comment]
Good point. I will try that. I will remove the file from the unreal engine
files and then put it back later.

**Comment by steveward** [comment]
👍🏻 Let me know if that gets things working.

**Comment by Gastropied** [comment]
It seems to be a constant issue.
However. I do believe the issue is "non existent". GitHub believes the
files have been changed when in fact they have not. And pushing them
committing them to origin pretty much solves the issue. I know for a fact
the files weren't changed so it doesn't matter if they get pushed.

I also found .gitignore files to add to the unreal engine files and it also
fixes the pop up blocking issue.
